doc/protocol.md

```diff
@@ -355,3 +355,62 @@ errors, which means that the same fragment could very well be received multiple
 Relays may forward fragments even if they arrive out of order.
 
 
+# Common Encoding For Media over QUIC Streams
+
+Media data can be sent over QUIC Streams in one of the following ways of grouping the media
+
+- One Stream per Group of Pictures/Group of Obejcts (like with WARP)
+- One Stream per frame (like with RUSH)
+- One Stream per Track/Rendition (like with QUICR)
+
```

Comment:
I understand why you will want commonality, but the "single stream per track/rendition" is harder, or at least is harder today. The initial prototype used the same QUIC stream for control and for data, and that's definitely a different format than using one bidirectional stream for control and an unidirectional stream for data. I guess that we could change that, but maybe that should be a separate PR.

Reply:
Also, do we have a definition of "track/rendition"?